### DISCUSSION TITLE
Super + Enter won't open Kitty Terminal

### DISCUSSION THREAD

**Discussion by captainvera** [regolith-linux/regolith-desktop#1083] [Q&A] [answered]
Hey there! I have an (almost) fresh install of ubuntu 22.04 LTS with regolith 3.2 installed.

Everything is working great except when I changed my default terminal from `gnome-terminal` to `kitty`.

I did `update-alternatives --config x-terminal-emulator` as suggested on regolith instructions but now nothing happens when I do `super` + `enter`.

Other keybinds still work great and I can launch kitty from ilia (`super` + `space` + kitty) but it's annoying that the shortcut doesn't work.

When I do `super` + `enter` nothing happens, how can I debug this?

**Comment by ingsme** [comment, marked as answer]
This sounds like a problem with update-alternatives.
- Try starting `x-terminal-emulator` from command line
- Check output of `update-alternatives --query x-terminal-emulator`

**Reply by captainvera** [reply]
Thank you!!

Your hints led me to the fact that `x-terminal-emulator` was returning `command not found` which, after some digging and understanding all the symlinks it uses, pointed me to the fact that this was pointing to `kitty.app` and not to the binary :man_facepalming: 

After correcting this it is now working seamlessly!

For anyone that stumbles upon this, this sets it up correctly:

```
sudo update-alternatives --install /usr/bin/x-terminal-emulator x-terminal-emulator `which kitty` 50
sudo update-alternatives --config x-terminal-emulator
```
